src/Microsoft.Restier.Publishers.OData/Routing/HttpConfigurationExtensions.cs

```diff
@@ -60,8 +60,17 @@ public static class HttpConfigurationExtensions
                     batchHandler.ApiFactory = apiFactory;
                 }
 
+
+                // Customized path handler should be added in ConfigureApi as service
+                // Allow to handle URL encoded slash (%2F), and backslash(%5C) with customized handler
+                var handler = api.Context.GetApiService<IODataPathHandler>();
+                if (handler == null)
+                {
+                    handler = new DefaultODataPathHandler();
```

Comment:
It's better to register the default type `DefaultODataPathHandler` to the container.